README.md

```diff
@@ -1 +1,33 @@
-# github-flow
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# github-flow
+
+> This is an example of github flow. Add new feature branch and make commits.
+
+
+## Built With
+
+- Js
+- Atom Git
+
```

Comment:
I am not sure you used any Js in tihis project, so you can take it out and consider adding languages and technologies that you actually used for this project, like HTML, CSS, GIT etc